packages/core/src/scss/components/foundation/buttons/_buttons.scss

```diff
@@ -60,4 +60,10 @@
     .button--slab {
         width: 100%;
     }
+
+    .link--disabled {
```

Comment:
Is it possbile to use L31 to avoid making a new class?

Reply:
No, It is impossible, because of [`<a />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) doesn't have a 'disabled' state.